lib/build.js

```diff
@@ -266,10 +285,29 @@ export default {
 
       let stdout = '';
       let stderr = '';
+      let liveErroMatchQueue = '';
+      let liveErroMatchHandle = target.liveErrorMatch ? function (d) {
```

Comment:
Well yes, 
If i have liveErrorMatch i don't want to parse again when finished. This actually does not make sense.
For example in my case i run my app through iOS simulator. When ending the session it means i either stopped to build by choice(esc) or closed the simulator. I don't think at that point i need to parse the stdout/stderr again.

Which means with this target (Titanium Mobile on iOS simulator) i want liveErrorMatch and not "end" error Match.
But i still don't want it when building with make or cmake for example.
So i guess it is a target parameter